packages/zmarkdown/client/index.js

```diff
@@ -0,0 +1,71 @@
+export const plugins = {}
+export let defaultType
+
+export function use (obj) {
+  if (!obj || (typeof obj !== 'object')) {
```

Comment:
What is `!obj` for?

Reply:
Is to check if the object exist (not null not undefined, etc.)

Reply:
I have do that because `typeof obj` return `"object"`.